lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -454,52 +456,8 @@ void SpecialFunctionHandler::handleMemalign(ExecutionState &state,
         0, "Symbolic alignment for memalign. Choosing smallest alignment");
   }
 
-  // Try to split up the memory allocation
```

Comment:
Removed all of this for handleAllocPmem

Reply:
Wait... are we intercepting calls to posix_memalign as special functions? RECIPE uses posix_memalign and that needs to go through jemalloc, not through SpecialFunctionHandler.

Reply:
Yeah, KLEE has a handleMemalign function. Always has.